tests/playwright/utilities/terminal.js

```diff
@@ -0,0 +1,63 @@
+import { execSync } from 'node:child_process';
+import path from 'path';
+
+export default class Terminal {
+	/*
+	* Activate or Deactivate experiments
+	* @param action {string} - activate | deactivate
+	* @param experiments {string} - comma separated list of experiments. e.g. container,nested-elements,'grid container'
+	*
+	* @example experiments( 'activate', 'container,nested-elements,grid container' )
+	* @example experiments( 'deactivate', 'container,nested-elements,grid container' )
+	*/
+	async experiments( action, experiments ) {
+		if ( action !== 'activate' && action !== 'deactivate' ) {
```

Comment:
@Barcov Rather create an array of "whitelisted" actions and check if the `action` is in that array. You can save it in the `enums` folder maybe.